crates/fluvio-extension-common/src/output/table.rs

```diff
@@ -57,20 +56,18 @@ where
 
         // Create the table
         let mut table = Table::new();
-        let mut format = *format::consts::FORMAT_CLEAN;
-        let pad_left = if indent { 5 } else { 1 };
-        format.padding(pad_left, 1);
-        table.set_format(format);
 
         // add header
-        table.set_titles(header);
+        table.set_header(header);
 
         // add rows
         for row in content {
             table.add_row(row);
         }
 
+        table.load_preset(comfy_table::presets::NOTHING);
+
         // print table to stdout
-        table.printstd();
+        println!("{table}");
```

Comment:
I think that we should use a custom trait instead of just calling `println!` and also we should [use](https://github.com/Nukesor/comfy-table/issues/7#issuecomment-648126461) `indent` there